packages/plugins/documentation/server/services/utils/get-plugins-that-need-documentation.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const getPluginsThatNeedDocumentation = (config) => {
+  // Default plugins that need documentation generated
+  const defaultPlugins = ['upload', 'users-permissions'];
+
+  // User specified plugins that need documentation generated
+  const userPluginsConfig = config['x-strapi-config'].plugins;
+
+  if (userPluginsConfig === null) {
+    // The user hasn't specified any plugins to document, use the defaults
+    return defaultPlugins;
+  }
+
+  if (userPluginsConfig.length) {
+    // The user has specified certain plugins to document, use them
+    return userPluginsConfig;
```

Comment:
So if the user still wants to document upload in addition to his own plugin, he'll have to include it in his array?

Reply:
Yes that's correct, it doesn't build upon the default values.